dojo/forms.py

```diff
@@ -380,7 +380,8 @@ class ImportScanForm(forms.Form):
                          ("Wpscan", "Wpscan"),
                          ("Sslscan", "Sslscan"),
                          ("JFrog Xray Scan", "JFrog Xray Scan"),
-                         ("Sslyze Scan", "Sslyze Scan"),
+                         ("SSLyze 2 Scan (XML)", "SSLyze 2 Scan (XML)"),
```

Comment:
changing scanner id/names will break existing api scripts. it might work if you just changed the 2nd part of the tuple, but haven't tried that
```
("Sslyze Scan", "Sslyze Scan v2 (xml)"),
```

Reply:
I like to be explicit, so it's clear for users what is expected and what get's delivered. Couldn't we mention the breaking change in the release notes? So users of SSLyze will be made aware it's time to switch to a newer version which is more reliable and delivers better results.

If that it's not an option for you, I of course can change it and make it backwards compatible.

Reply:
I don't like breaking things, having to deal with github issues, questions on slack, release notes etc.

Reply:
Alright, you might have more experience with this in open source projects. I will change it making it backwards compatible.

Reply:
Merci :-)